server/views/pages/bookedtoday/arrivals/searchforexisting/singleExistingRecordFound.njk

```diff
@@ -13,12 +13,18 @@
         {{ govukInsetText({
             html: 'If you think this is the wrong record, you can <a class="govuk-body govuk-link govuk-link--no-visited-state" data-qa="search-instead" href="/prisoners/' + data.id + '/search-for-existing-record/new">search for a record using different details</a> or <a class="govuk-body govuk-link govuk-link--no-visited-state" data-qa="create-new" href="/prisoners/' + data.id + '/review-per-details/new">create a new record</a>.'
         }) }}
-        {{ govukButton({
-            classes: "govuk-button govuk-!-margin-bottom-6",
-            href: "/prisoners/" + data.id + "/start-confirmation",
-            text: "Continue",
-            attributes: {'data-qa': 'continue'}
-        }) }}
+
+        <form method="post" action={{ "/prisoners/" + data.id + "/search-for-existing-record/record-found" }}>
```

Comment:
I think we can remove the action and it will inherit the url from where its hosted from?

Reply:
Did try and do a quick google and couldn't find anything very opinionated. 

In this case, I like the specificity since this page could be loaded from any route (technically), but it will always POST data to one defined resource.